### ISSUE TITLE
Support for HistGradientBoostingClassifier

### ISSUE THREAD

**Issue by DiTo97** [dmlc/treelite#434]
Hi @hcho3,

As the title suggests, is support for Scikit-learn's [hist gradient boosting classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html) on the roadmap as those already implemented [here](https://treelite.readthedocs.io/en/latest/tutorials/import.html?highlight=Scikit#importing-scikit-learn-models)?

If not, is there a reason why? Or the wrapper to the "standard" gradient boosting classifier is enough to wrap its hist variant as well?

**Comment by hcho3** [comment]
No, there isn't a particular reason for not supporting HistGradientBoostingClassifier. I just haven't gotten around implementing it yet

**Comment by DiTo97** [comment]
> No, there isn't a particular reason for not supporting HistGradientBoosting. I just haven't gotten around implementing it yet

That was fast! Thank you.

In my experience, it is significantly more performant than standard gradient boosting with very large datasets with lots of features. Do you have any guidelines? I may be interested to implement it myself with the builder

**Comment by hcho3** [comment]
@DiTo97 I filed #444 to add support for HistGradientBoosting.

**Comment by DiTo97** [comment]
Thank you @hcho3,

I will keep an eye on that PR. As for #445, how big of a re-work will it take to make `global_bias` hold a vector rather than a scalar?

**Comment by hcho3** [comment]
@DiTo97 It will be a lot of work, since we need to add a new field for vector `global_bias`.
